### ISSUE TITLE
Make UWPX accessible to narrator and other screen readers

### ISSUE THREAD

**Issue by MarcoZehe** [UWPX/UWPX-Client#61]
## Bevor you create a new issue:

```
- [ x ] I searched for similar issues and did not find one
- [ x ] I'm using the latest version available in the [Windows Store](https://www.microsoft.com/store/apps/9NW16X9JB5WV)
```

## I'm submitting a...:

- Bug report (I searched for similar issues and did not find one) 
 

## Current behavior:

UWPX does not expose any useful information to assistive technologies such as Narrator (invoked with CTRL+WindowsKey+Enter) or other screen readers such as [NVDA](https://www.nvaccess.org). Buttons are not labeled, and other controls also expose no useful information. Right now, a blind user can't even log into a server.

## Expected behavior:

Labeled controls and otherwise accessible application. More information: [Accessibility main page for UWP apps](https://docs.microsoft.com/en-us/windows/uwp/design/accessibility/accessibility), and especially [Expose basic accessibility information](https://docs.microsoft.com/en-us/windows/uwp/design/accessibility/basic-accessibility-information).

## Minimal reproduction of the problem with instructions:
1. [Get started with Narrator](https://support.microsoft.com/en-us/help/22798/windows-10-complete-guide-to-narrator).
2. Start Narrator and UWPX.
3. Tab around the interface and listen to what you hear, and don't hear.

## Environment:

```
App Version(s): 
v.0.15.0 <!-- Which version of the App are you using (e.g. v.0.2.0) -->

Windows 10 Version Number: <!-- https://en.wikipedia.org/wiki/Windows_10_version_history -->
- [ x ] 1809
- [ ] 1803
- [ ] 1709
- [ ] 1703
- [ ] 1607
- [ ] 1511
- [ ] 1507
- [ ] Insider Build (build number: )
- [ ] Misc:

Device form factor:
- [ x ] Desktop
- [ ] Mobile
- [ ] Xbox
- [ ] Surface Hub

Where did you got the APP from?
- [ x ] [Windows Store](https://www.microsoft.com/store/apps/9NW16X9JB5WV)
- [ ] Self-build, using a provided release source
- [ ] Self-build, repo cloned at [dd.mm.yyy] <!-- When did you clone the repo! -->
- [ ] Misc, got it from... <!-- Please tell us your source! -->

```

**Comment by COM8** [comment]
Thanks for opening the issue.
Will add it to the UI Overhaul (#50) todo list.
